lib/remap-functions/src/to_int.rs

```diff
@@ -12,7 +12,7 @@ impl Function for ToInt {
     fn parameters(&self) -> &'static [Parameter] {
         &[Parameter {
             keyword: "value",
-            accepts: |v| matches!(v, Value::Integer(_) | Value::Float(_) | Value::Bytes(_) | Value::Boolean(_) | Value::Timestamp(_) | Value::Null),
+            accepts: crate::util::is_scalar_value,
```

Comment:
`is_scalar_value` does not return true for `Value::Timestamp(_)`, so I think we'll have to revert this change? (that function is poorly named, it started out correctly, but has since diverged from its name a bit)